build/charts/antrea/values.yaml

```
@@ -105,6 +105,8 @@ nodePortLocal:
   portRange: "61000-62000"
 
 antreaProxy:
+  # -- To enable AntreaProxy, you need to set "enable" to true.
```

```suggestion
  # -- To disable AntreaProxy, set this to false.
```

It makes more sense since AntreaProxy is enabled by default.

---

Done